clang/test/Analysis/region-store.c

```diff
@@ -1,6 +1,6 @@
 // RUN: %clang_analyze_cc1 -analyzer-checker=core,unix,debug.ExprInspection \
 // RUN:    -verify -analyzer-config eagerly-assume=false -std=c99 %s \
-// RUN:    -Wno-implicit-function-declaration
+// RUN:    -Wno-implicit-function-declaration -w
```

Comment:
`-Wno-implicit-function-declaration` can now be dropped, it's implied by `-w` ^^